qiskit_experiments/composite/composite_experiment.py

```diff
@@ -65,3 +65,9 @@ def _add_job_metadata(self, experiment_data, job, **run_options):
             sub_exp = self.component_experiment(i)
             sub_data = experiment_data.component_experiment_data(i)
             sub_exp._add_job_metadata(sub_data, job, **run_options)
+
+    def _postprocess_transpiled_circuits(self, circuits, backend, **run_options):
+        for expr in self._experiments:
+            if not isinstance(expr, CompositeExperiment):
```

Comment:
Does that mean you're assuming not nesting of `CompositeExperment`s ?

Reply:
Indeed. I can implement a more advanced logic, but I think that's really something that should be done in the `CompositeExperiment` side.